netbout-web/src/main/java/com/netbout/rest/bout/TkAttach.java

```diff
@@ -133,7 +122,27 @@ public Response act(final Request req) throws IOException {
             throw new RsFailure(ex);
         }
         FileUtils.forceDelete(temp);
-        throw new RsForward(new RsFlash(msg.toString()));
+        bout.messages().post(msg.toString());
+        return new RsForward(new RsFlash(msg.toString()));
```

Comment:
@pinaf Actually, you're right, there seems to be a difference in behavior. The thrown `RsForward` is handled [here](https://github.com/yegor256/netbout/blob/master/netbout-web/src/main/java/com/netbout/rest/bout/FkBout.java#L114).

Then, should we keep try..catch block in the test?